.github/workflows/npmtest.yml

```
@@ -0,0 +1,35 @@
+on:
+  workflow_call:
+
+name: Run npm test and lint
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - uses: shogo82148/actions-setup-redis@v1
+      with:
+        redis-version: '6.2'
+
+    - name: Set up Outfunnel Registry
+      run: |
+        echo "@outfunnel:registry=https://registry.outfunnel.com/" > .npmrc
```

```suggestion
        echo "@outfunnel:registry={{secrets.OF_REGISTRY_URL}}/" > .npmrc
```
SR - I suggest not exposing this URL.